courses/sc-rna-seq-snap-repo/README.md

```diff
@@ -0,0 +1,84 @@
+
+<br />
+<!-- badges: start -->
+<a href="https://opensource.org/license/bsd-2-clause/"><img src="https://img.shields.io/badge/open-source-logo.png" alt="License: BSD 2-Clause License" /></a>
+<a href="https://www.repostatus.org/#active"><img src="https://www.repostatus.org/badges/latest/active.svg" alt="The project has reached a stable, usable state and is being actively developed." /></a>
+<a href="https://github.com/stjude-dnb-binfcore/sc-rna-seq-snap/graphs/traffic"><img src="https://img.shields.io/badge/traffic" alt="Monthly Downloads" /></a>
+<a href="https://github.com/stjude-dnb-binfcore/sc-rna-seq-snap/forks"><img src="https://img.shields.io/badge//forks" alt="Total Forks" /></a>
+<!-- badges: end -->
+
+<br />
+</div>
+
+
+
+# Single cell RNA Seq Snap workflow (ScRNASeqSnap) Workshop
+
+The [Single cell RNA Seq Snap workflow (ScRNASeqSnap)](https://github.com/stjude-dnb-binfcore/trainings/tree/main/courses/sc-rna-seq-snap-repo/) workshop is an introduction to the upstream and downstream analyses modules for pre-processing and analyzing single cell and single nuclei RNA (sc/snRNA) data from 10X sequencing technology. This workshop is an initiative of our team at the [Bioinformatics Core at the Department of Developmental Neurobiology at the St. Jude Children's Research Hospital](https://www.stjude.org/research/departments/developmental-neurobiology/shared-resources/bioinformatic-core.html)
+
+In this workshop, we will discuss tools, methods, and reports (including results and plots) generated by the [ScRNASeqSnap](https://github.com/stjude-dnb-binfcore/sc-rna-seq-snap), a powerful and dynamic pipeline developed by our team at the [DNB Bioinformatics Core](https://github.com/stjude-dnb-binfcore).
+
+
+
+| Audience | Computational Skills | Prerequisites | Duration |
+:----------|:----------|:----------|:----------|
+| Biologists/Bioinformaticians | Beginner/Intermediate | Shell basics | 1 h|
+
+
+## Learning Objectives
+
+1. Overview of Tools and Methods in the Repo
+2. Best Practices for automation and reproducibility: (a) documentation and usability, (b) automation, and (c) containers
+3. Reviewing Results, Plots, and Reports
+
+
+## Target Audience
+The course is intended for cancer informatics tool developers, particularly those creating tools as a part of the [Department of Neurobiology at the St. Jude Children's Research Hospital](https://www.stjude.org/research/departments/developmental-neurobiology.html), but also for wet lab biologists that are interested in exploring the capabilities of the `ScRNASeqSnap` pipeline.
+
+> These materials were developed for a trainer-led workshop, but are also amenable to self-guided learning.
+
+## Lessons 🚧 🚧 🚧
+* [sc-rna-seq-snap-repo-2024-12](https://github.com/stjude-dnb-binfcore/trainings/tree/main/courses/sc-rna-seq-snap-repo/lectures/sc-rna-seq-snap-repo-2024-12.pdf)
+
+
+## Access to the workshop recording 🚧 🚧 🚧
+
+St. Jude employees can have access to the [recording from the workshop]().
```

Comment:
Maybe this part should not be included until after the workshop, to encourage in person attendance

Reply:
Fixed here 0bf688880110c2317e9d1fe4e2c73373274bca5d